chart/epinio/templates/NOTES.txt

```diff
@@ -1,15 +1,7 @@
-
-{{- if .Values.global.domain }}
-You can get the IP address of your Ingress controller by passing this command:
-`kubectl describe svc traefik --namespace traefik | grep Ingress | awk '{print $3}'`
-
-Make sure your domain points to the IP address of your Ingress controller (1.2.3.4).
```

Comment:
Do we really want to remove this message? because the user has to think about updating its DNS with the correct IP. 
When we talked about it this morning, I though we wanted to remove the `kubectl` command and just say something like `Get the IP address of your Ingress controller and update your DNS accordingly.`